app/app.pro

```diff
@@ -0,0 +1,18 @@
+TEMPLATE = app
+TARGET = telescope
+
+load(ubuntu-click)
+
+QT += qml quick dbus
+CONFIG += c++11
+
+DESTDIR = lib/arm-linux-gnueabhif/qt5/qml/org/asteroid/syncservice/asteroidsyncserviceplugin
```

Comment:
If I understand right, the telescope binary is installed in the asteroidsyncserviceplugin directory, isn't it? Any reason for that?
Also, I see here that you hardcode "arm-linux-gnueabhif" and in telescope-wrapper you also handle "x86_64-linux-gnu". Can you maybe explain the rationale between this?

Reply:
![image](https://user-images.githubusercontent.com/35935077/62425566-48facd80-b6db-11e9-9cb1-dddf8a92db40.png)
Here you can see the structur in the click paket.
In the wrapper the "arm-linux-gnueabhif" is hard coded, because i need to set different QML2_IMPORT_PATH paths. Depending if i run the program on desktop or mobile phone.

All in all i'm not happy with this hardcoded path's, but i don't know a better way.

Reply:
It take a look on the app/app.pro and the "DESTDIR" was left over from the project template "starfish/ubports" branch and not necessary.
I removed it.
Thanks a lot for the great review.